dir.props

```diff
@@ -114,7 +114,7 @@
   <!-- Dependency auto-upgrade configuration -->
   <ItemGroup>
     <DependencyBuildInfo Include="CoreFx">
-      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/$(DependencyBranch)</RawUrl>
+      <RawUrl>$(BaseDotNetBuildInfoUrl)corefx/dev/api</RawUrl>
```

Comment:
I think one thing we can do is to split out the property value into a new branchinfo.props that we add to the gitignore file so they will only ever be updated if we force add them. (http://stackoverflow.com/questions/8006393/force-add-despite-the-gitignore-file). That should make merges clean.